hoptimator-util/src/main/java/com/linkedin/hoptimator/util/PropertyUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.hoptimator.util;
+
+import java.util.Properties;
+
+public final class PropertyUtils {
+
+  public static final String HOPTIMATOR_CONFIG_DOMAIN = "hoptimator.config";
```

Comment:
Instead of prefixes (e.g. `venice.foo=bar`) inside a common file (`hoptimator.config`) would it be useful to break out each driver's config into separate files?

```
venice.config:
  foo=bar
```

I'm thinking each driver could do smth like `loadConfigs("venice.config")` to get driver-specific configuration?

Reply:
We could but I was thinking there would be a lot of overlap for certain fields between drivers (Like cluster or environment type properties).

ConfigMaps do let you nest properties, do you think something like that would be better?
Example:
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: hoptimator-operator
data:
  env: "dev"
  cluster: "my-cluster"

  venice.properties: |
    venice.router=localhost:823
```

Reply:
I see. I don't think that's any better than prefixes.

What does "domain" mean here?

Reply:
Just a prefix, I'm getting rid of that bit since I don't like that I am loading these into System properties right now, I will load these as needed per driver